RiskAssessmentApi.yaml

```
@@ -248,9 +376,195 @@ components:
       - directionDegrees
       - speedMetersPerSecond
 
+    ObstacleProximityRiskThresholds:
+      type: object
+      description: A collection of thresholds for setting obstacle proximity risk level
+      properties:
+        minimumDistanceMeters:
+          type: number
+          format: double
+          minimum: 0.0
+          description: The threshold for the minimum distance to any obstacle in meters
+        obstacleCount:
+          type: number
+          format: double
+          minimum: 0
+          description: The threshold for the number of obstacles within the minimum distance
+      required:
+      - minimumDistanceMeters
+      - buildingCount
```

"buildingCount" is not a property, presumably a rename miss
```suggestion
      - obstacleCount
```

---

Also, how does the swagger editor warn about extra descriptions, but not about misaligned required fields?!